diff --git a/src/helpers/AppContext.ts b/src/helpers/AppContext.ts
index b77ebc4..c527d37 100644
--- a/src/helpers/AppContext.ts
+++ b/src/helpers/AppContext.ts
@@ -1,3 +1,4 @@
+//@ts-nocheck
 import * as _ from "lodash";
 import { AppContext, Game, MapPool, Round } from "../types-interfaces/Interfaces";
 import { maps, modeAbbreviations } from "./MapMode";
diff --git a/src/helpers/MapGeneration.ts b/src/helpers/MapGeneration.ts
index 9a2ca19..1fca8c5 100644
--- a/src/helpers/MapGeneration.ts
+++ b/src/helpers/MapGeneration.ts
@@ -1,3 +1,4 @@
+//@ts-nocheck
 import _ from "lodash";
 import { Game, MapPool, Round } from "../types-interfaces/Interfaces";
 import { GenerateMode, Map, Mode } from "../types-interfaces/Types";
diff --git a/src/helpers/MapMode.ts b/src/helpers/MapMode.ts
index 60e7755..a6cdf58 100644
--- a/src/helpers/MapMode.ts
+++ b/src/helpers/MapMode.ts
@@ -1,4 +1,4 @@
-import { Mode } from "../types-interfaces/Types";
+import type { Mode } from "../types-interfaces/Types";
 
 export const modeAbbreviations = ['tw', 'sz', 'tc', 'rm', 'cb'] as const;
 
diff --git a/src/types-interfaces/Interfaces.ts b/src/types-interfaces/Interfaces.ts
index b49641f..dd44717 100644
--- a/src/types-interfaces/Interfaces.ts
+++ b/src/types-interfaces/Interfaces.ts
@@ -1,4 +1,4 @@
-import { Counterpick, PlayStyle, Map, Mode } from "./Types";
+import type { Counterpick, PlayStyle, Map, Mode } from "./Types";
 
 export interface AppContext {
     mapPool: MapPool
